calabash/Classes/FranklyServer/Routes/LPReflectionRoute.h

```diff
@@ -0,0 +1,6 @@
+#import <Foundation/Foundation.h>
+#import "LPRoute.h"
+
+@interface LPReflectionRoute : NSObject<LPRoute>
```

Comment:
Just out of curiosity, is there an advantage of using a `@protocol` vs inheritance? I know this is how all of the routes are made, strictly curious.